### DISCUSSION TITLE
38. CORS에서 쿠키를 사용할 때 어떤 설정이 필요한가요?

### DISCUSSION THREAD

**Discussion by linirini** [jewelry-box/cs-interview#39] [General]
.

**Comment by jongmee** [comment]
서버에서 Access-Control-Allow-Credentials를 true로 설정하고 Access-Control-Allow-Origin, Access-Control-Allow-Methods, Access-Control-Allow-Headers에 와일드카드를 사용하면 안된다.

**Reply by cutehumanS2** [reply]
(참고) Access-Control-Allow-Methods와 Access-Control-Allow-Headers는 CORS에서 쿠키 사용 여부와 관계없이 원래부터 와일드카드(*)를 지정할 수 없다.
